subsys/mgmt/hawkbit/hawkbit.c

```diff
@@ -974,10 +1144,48 @@ static bool send_request(enum http_method method, enum hawkbit_http_request type
 	return true;
 }
 
+#ifdef CONFIG_HAWKBIT_SET_SETTINGS_RUNTIME
+static int check_hawkbit_server(void)
```

Comment:
looks like this is configuring the default setting (Kconfig) instead?

Reply:
yes it is using them as a failback